buildSrc/src/main/groovy/airbyte-bulk-connector.gradle

```diff
@@ -138,9 +158,25 @@ class AirbyteBulkConnectorPlugin implements Plugin<Project> {
             dependsOn project.tasks.matching { it.name ==~ /(compile|spotbugs)[a-zA-Z]*Java/ }
         }
 
+        project.tasks.register('integrationTest', Test) {
+            description = 'Runs the integration tests.'
+            group = 'verification'
+            testClassesDirs = project.sourceSets.integrationTestJava.output.classesDirs
+            classpath = project.sourceSets.integrationTestJava.runtimeClasspath
+            useJUnitPlatform()
+            mustRunAfter project.tasks.check
+        }
+
+        project.tasks.named('build').configure {
```

Comment:
I feel like this is a weird thing to do, but I'm guessing the old cdk also did this so :shrug:

Reply:
It's weird, and normally the dependency should be on `check`, but there's a good reason here.

The reason for `build` in the legacy CDK is that the integration tests require the connector docker image, which puts a dependency on `assemble`, and `build == assemble + check` by definition. Since the point here is to also use the docker images, it makes sense to carry this over.